grpc/proto/th2_grpc_read_db/read_db.proto

```diff
@@ -57,7 +57,7 @@ message DbPullRequest {
 
 message DbPullResetState {
   google.protobuf.Timestamp after_date = 1;
-  google.protobuf.Timestamp afterTime = 2;
+  google.protobuf.Timestamp after_time = 2; // Only time units matter, value must contains '1970-01-01' date
```

Comment:
Could you check if this goes into generated stubs? At least for Java. If I remember correctly only comment before field will go to Javadoc

Reply:
![Screenshot from 2023-11-16 10-26-18](https://github.com/th2-net/th2-read-db/assets/46124551/afb2b617-554a-44d4-a9f4-3626a2ba7626)